endshipper.go

```diff
@@ -0,0 +1,71 @@
+package easypost
+
+import (
+	"context"
+	"net/http"
+)
+
+// ListEndShipperResult holds the results from the list EndShippers API.
+type ListEndShipperResult struct {
+	EndShippers []*Address `json:"endshippers,omitempty"`
```

Comment:
Do we not want to make a specific class for EndShipper? I know it's basically just an Address based on the properties, but for the future? Also to prevent someone from using an EndShipper as a normal address?

Reply:
Per internal discussion, the answer is we are going to `address` for now. cc @Justintime50

Reply:
Well, why not split them up? What did we do with Java/C#?

Reply:
We use inheritance in C# and Java, but unfortunately in Go there is no Inheritance

Reply:
Hmm.. what should we do? Run the risk of objects changing over time or reuse Address because EndShipper is literally an Address?

Reply:
I agree with Nate, if somehow in the future EndShipper properties diverge from Address, then it would be better to have EndShipper struct itself

Reply:
I say diverge and make an explicit class.

I know I was a big proponent of base classes and inheritance for C# and Java, but honestly, Address has been the only thing so far that we've used it for, so it's largely been unnecessary.

Reply:
Let's split em up then!

Reply:
We have decided not to split em up